nomad/mock/acl.go

```diff
@@ -4,11 +4,13 @@
 package mock
 
 import (
+	"bytes"
 	"crypto/rand"
 	"crypto/rsa"
 	"crypto/x509"
 	"encoding/pem"
 	"fmt"
+	"html/template"
```

Comment:
I think?

```suggestion
	"text/template"
```

Reply:
Yeah, makes sense. This was auto-imported. Thanks!